src/library/util.cpp

```diff
@@ -1097,6 +1103,19 @@ void initialize_library_util() {
     initialize_int();
     initialize_char();
     initialize_bool();
+
+    sstream out;
+
+    out << LEAN_VERSION_MAJOR << "."
+        << LEAN_VERSION_MINOR << "." << LEAN_VERSION_PATCH;
+    if (std::strcmp(LEAN_GITHASH, "GITDIR-NOTFOUND") == 0) {
+        if (std::strcmp(LEAN_PACKAGE_VERSION, "NOT-FOUND") != 0) {
+            out << ", package " << LEAN_PACKAGE_VERSION;
+        }
+    } else {
+        out << ", commit " << std::string(LEAN_GITHASH).substr(0, 12);
+    }
+    g_version_string = new std::string(out.str());
 }
```

Comment:
It seems `g_version_string` is not being deleted during finalization.

Reply:
Fixed, thanks